### ISSUE TITLE
Accessibility - Add support for aria-expanded="true/false" button

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluent-ui-react#1414]
### Problem description
There is needs that sometimes button can have aria-expanded property, which need be updated based on the button state.

### Proposed solution
We could do it similarly like Toggle button behavior. It should be discussed. :)

**Comment by jurokapsiar** [comment]
users can provite custom aria props if there is a need, closing
